samples/matter/closure/Kconfig

```
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+mainmenu "Matter nRF Connect Closure Example Application"
+
+# Sample configuration used for Thread networking
+if OPENTHREAD
+
+choice OPENTHREAD_NORDIC_LIBRARY_CONFIGURATION
+	default OPENTHREAD_NORDIC_LIBRARY_FTD
+endchoice
+
+choice OPENTHREAD_DEVICE_TYPE
+	default OPENTHREAD_FTD
+endchoice
+
+endif # OPENTHREAD
+
+config CHIP_WIFI
+	default y if BOARD_NRF7002DK_NRF5340_CPUAPP
```

We are going to deprecate nRF7002DK soon, so there is no point to add it to new samples. On the other hand we have nRF54LM20DK + nRF7002 EB2 shield configuration.
```suggestion
	default y if SHIELD_NRF7002EB2
```